[2025-01-29T10:54:09.594+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-29T10:54:09.660+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.product_Load_redshift manual__2025-01-29T10:53:39.199622+00:00 [queued]>
[2025-01-29T10:54:09.696+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.product_Load_redshift manual__2025-01-29T10:53:39.199622+00:00 [queued]>
[2025-01-29T10:54:09.698+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 3
[2025-01-29T10:54:09.758+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): product_Load_redshift> on 2025-01-29 10:53:39.199622+00:00
[2025-01-29T10:54:10.017+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1480) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-29T10:54:10.040+0000] {standard_task_runner.py:72} INFO - Started process 1505 to run task
[2025-01-29T10:54:10.074+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'product_Load_redshift', 'manual__2025-01-29T10:53:39.199622+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpcbsxygbc']
[2025-01-29T10:54:10.125+0000] {standard_task_runner.py:105} INFO - Job 25: Subtask product_Load_redshift
[2025-01-29T10:54:10.908+0000] {task_command.py:467} INFO - Running <TaskInstance: main_dag.product_Load_redshift manual__2025-01-29T10:53:39.199622+00:00 [running]> on host ae52aba2dca2
[2025-01-29T10:54:12.740+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='itsMe!!' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='product_Load_redshift' AIRFLOW_CTX_EXECUTION_DATE='2025-01-29T10:53:39.199622+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-29T10:53:39.199622+00:00'
[2025-01-29T10:54:12.758+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-29T10:54:17.305+0000] {logging_mixin.py:190} INFO - Data loaded successfully into table!
[2025-01-29T10:54:17.311+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-29T10:54:17.418+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-29T10:54:17.419+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=product_Load_redshift, run_id=manual__2025-01-29T10:53:39.199622+00:00, execution_date=20250129T105339, start_date=20250129T105409, end_date=20250129T105417
[2025-01-29T10:54:17.722+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-29T10:54:17.840+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-29T10:54:17.848+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
